feature-libs/product-configurator/rulebased/components/price/configurator-price.component.ts

```diff
@@ -41,7 +45,11 @@ export class ConfiguratorPriceComponent {
    * @return {string} - total price formula
    */
   get priceTotal(): string {
-    return '+ ' + this.formula.priceTotal?.formattedValue;
+    if (this.formula.priceTotal && this.formula.priceTotal.value >= 0) {
```

Comment:
I think the condition should be `this.formula.priceTotal.value >= 1` and not `this.formula.priceTotal.value >= 0`.
The end-user wants to see price when it is cheaper or more expensive but not when the price equals zero.
Plz, add a test for zero case.